app/_components/commodities/components/GridBodyItem.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import { formatThousands } from '@/app/_hooks/textFormatting';
+import { GridItem, SimpleGrid } from '@chakra-ui/react';
+import { ICommodityFormResponse } from '@/app/_types/commodity';
+import selectColor from '@/app/_hooks/fontColorSelector';
+import useColorMode from '@/app/_hooks/useColorMode';
+import { calculateTimeDifference, renderStationTypeIcon } from '../helpers';
+
+interface IGridBodyItemProps {
+  commodity: ICommodityFormResponse;
+  isBuying: boolean;
+}
+
+const GridBodyItem: React.FC<IGridBodyItemProps> = ({
+  commodity,
+  isBuying,
+}) => {
+  const { isDark } = useColorMode();
+
+  return (
+    <SimpleGrid
+      columns={[5, 7]}
+      width="100%"
+      gap={8}
+      fontSize="sm"
+      _odd={{
+        background: `${selectColor(isDark, '')}`,
+      }}
+      paddingY={2}
+    >
+      <GridItem>
+        CR{' '}
+        {isBuying
+          ? formatThousands(commodity.sellPrice)
+          : formatThousands(commodity.buyPrice)}
+      </GridItem>
+      <GridItem textAlign="right" paddingRight={8}>
```

Comment:
I'm not sure what the thinking is here with alignment but the content cells alignment doens't match the headers and it looks out of whack, can the alignment be made consistent

Reply:
fixed:

![Screenshot 2023-08-31 at 10-15-48 EDPN - Commodities](https://github.com/ed-pilots-network/frontend/assets/67390534/cb90282a-fa27-46ee-ad78-10c34b424fef)